packages/block-library/src/page-list/convert-to-links-modal.js

```diff
@@ -32,16 +32,14 @@ export function ConvertToLinksModal( { onClick, onClose, disabled } ) {
 			</p>
 			<div className="wp-block-page-list-modal-buttons">
 				<Button
-					// TODO: Switch to `true` (40px size) if possible
-					__next40pxDefaultSize={ false }
+					__next40pxDefaultSize
 					variant="tertiary"
 					onClick={ onClose }
 				>
 					{ __( 'Cancel' ) }
 				</Button>
 				<Button
-					// TODO: Switch to `true` (40px size) if possible
-					__next40pxDefaultSize={ false }
+					__next40pxDefaultSize
```

Comment:
This can be seen, while you add page-list block and try to convert page-list to navigationLinks

| Before | After     |
| :-------- | :------- |
| <img width="1429" alt="page-list-1-before" src="https://github.com/user-attachments/assets/8e3b8205-d2a2-435b-8459-fa9f0464d8a0"> | <img width="1429" alt="page-list-1-after" src="https://github.com/user-attachments/assets/07c01f93-677d-426a-9685-f99b82835ce1"> |